test/integration/Cover/expireCover.js

```diff
@@ -242,12 +238,6 @@ describe('expireCover', function () {
         { value: amount },
       );
 
-    const allocationsAfter = await stakingPool1.getActiveAllocations(productId);
-    const { totalActiveCoverInAsset: activeCoverAfter } = await cover.activeCover(coverAsset);
-
-    // stays the same because same amount expired and was allocated
-    expect(sum(allocationsBefore)).to.be.equal(sum(allocationsAfter));
-    expect(activeCoverBefore).to.be.equal(activeCoverAfter);
```

Comment:
I'm not 100% certain but it seems its possible that these expectations no longer hold true due to the dynamic pricing.

The cover amount from the first and second `buyCover` is the same but the token price is different.